internal/core/config.go

```diff
@@ -201,27 +208,22 @@ func (cluster ClusterConfiguration) validateConfig() (errs errext.ErrorSet) {
 	if len(cluster.AvailabilityZones) == 0 {
 		missing("availability_zones[]")
 	}
-	if len(cluster.Services) == 0 {
-		missing("services[]")
+	if len(cluster.Liquids) == 0 {
+		missing("liquids[]")
 	}
-	//NOTE: cluster.Capacitors is optional
 
-	for idx, srv := range cluster.Services {
+	//NOTE: Liquids[].FixedCapacityConfiguration and Liquids[].PrometheusCapacityConfiguration are optional
+	for idx, srv := range cluster.Liquids {
 		if srv.ServiceType == "" {
-			missing(fmt.Sprintf("services[%d].id", idx))
+			missing(fmt.Sprintf("services[%d].service_type", idx))
```

Comment:
```suggestion
			missing(fmt.Sprintf("liquids[%d].service_type", idx))
```
Same for the other messages in this for-loop.